doc/api_ref/pubkey.rst

```diff
@@ -938,6 +1106,9 @@ Don't use this unless you know what you are doing.
 ISO-9796-2
 """"""""""
 
+The ISO-9796-2 padding schemes are used for EMV signatures. There
+is likely no reason to use it in other contexts.
+
```

Comment:
Perhaps `-v` on EMV signatures? I've never come across this, FWIW.

Reply:
This [EMV](https://en.wikipedia.org/wiki/EMV)

I expanded this out to "EMV contactless payment card system" so it's unambigious